simplq/src/components/pages/Home/Nav/Burger.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
 import React, { useState } from 'react';
```

Comment:
The warnings are still present, so keeping these for now, so that we won't forget.